stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.26.2
+version: 1.26.3
```

Comment:
We are adding a feature so lets bump the minor version here.
`1.27.0`

Reply:
Changed